generators/app/gulpfile.js

```diff
@@ -11,6 +11,7 @@
 'use strict';
 
 const path = require('path');
+const process = require('process');
```

Comment:
I could be mistaken but I don't think you have to require process. It's global https://nodejs.org/api/globals.html#globals_process

Not sure what the recommended best practice is